osquery/tables/networking/darwin/wifi_status.mm

```diff
@@ -12,20 +12,52 @@
 
 #include <osquery/core/system.h>
 #include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
 #include <osquery/tables/networking/darwin/wifi_utils.h>
+#include <osquery/utils/darwin/system_profiler.h>
 
 namespace osquery {
 namespace tables {
 
 QueryData genWifiStatus(QueryContext& context) {
   QueryData results;
   @autoreleasepool {
-    std::string interfaceName = "en0";
+    // Get the list of Wifi interfaces on the system.
```

Comment:
This looks to have been hanging around for a long time, but it's not used anywhere I can see.